### PR TITLE
docs: main page

### PR DESCRIPTION
**Pull request by manudeli**
## Description

**Related Issue:** Fixes # (issue_number)

chrome-capture-2024-12-23 (1)

Mobile size design also needs to be worked on.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update

## Checklist

- [ ] I have performed a self-review of my own code.
- [ ] My code is commented, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: docs/src/components/main.tsx [hunk 1]
```diff
@@ -42,26 +271,32 @@ const styles = stylex.create({
   root: {
     display: 'flex',
     flexDirection: 'column',
-    gap: 120,
-    paddingTop: 98,
   },
-  mainRoot: {
+  title1: {
```
Comment (jungpaeng) [inline docs/src/components/main.tsx:275]: Hello! Thank you so much for contributing to the document.

I have one suggestion, so I’m leaving a comment. The text “Declarative Overlay Pattern” overlaps with the background, which makes it difficult to read.

There might be several ways to fix this, but I think the easiest approach is to add an outline to the text using CSS like the example below. What do you think about adding an outline in this way?

If you have any better ideas, please feel free to share!

Comment (jungpaeng) [reply docs/src/components/main.tsx:275]:
```js
textShadow:
  '1px 1px 0 rgba(0, 0, 0, 0.1), -1px 1px 0 rgba(0, 0, 0, 0.1), 1px -1px 0 rgba(0, 0, 0, 0.1), -1px -1px 0 rgba(0, 0, 0, 0.1)',
```

Comment (manudeli) [reply docs/src/components/main.tsx:275]: It's small difference! I want to suggest below shadow. a little bit color with black

### TO-BE textShadow: '0 0 2px rgba(0, 0, 120, 0.1), 0 0 5px rgba(120, 0, 120, 0.1), 0 0 12px rgba(0, 0, 120, 0.1)',
image

### AS-IS textShadow: '0 0 2px rgba(0, 0, 120, 0.1), 0 0 5px rgba(120, 0, 120, 0.1), 0 0 12px rgba(0, 0, 120, 0.1)',
image

Comment (manudeli) [reply docs/src/components/main.tsx:275]: Is this textShadow okay? [4d46700](https://github.com/toss/overlay-kit/pull/101/commits/4d4670074e4832b0d12e9c57b87a40530bcd80b8)

Comment (manudeli) [reply docs/src/components/main.tsx:275]: I reflect your review! Thanks!

**Review by jungpaeng — APPROVED** [review]
LGTM!

**Pull request merged**